docs/help-and-support.md

```diff
@@ -0,0 +1,10 @@
+# Help and Support
+
+For Mattermost customers - please open a [support case](https://mattermost.zendesk.com/hc/en-us/requests/new) to ensure your issue is tracked properly.
+
+For Questions, Suggestions and Help - please find us on our forum at [https://forum.mattermost.org/c/plugins](https://forum.mattermost.org/c/plugins) 
+
+Alternatively, join our pubic Mattermost server and join the Integrations and Apps channel here: [https://community-daily.mattermost.com/core/channels/integrations](https://community-daily.mattermost.com/core/channels/integrations)
+
+To Contribute to the Mattermost project see [https://www.mattermost.org/contribute-to-mattermost/](https://www.mattermost.org/contribute-to-mattermost/)
```

Comment:
```suggestion
To contribute to the Mattermost project see [https://www.mattermost.org/contribute-to-mattermost/](https://www.mattermost.org/contribute-to-mattermost/).
```

Reply:
I should have edited some of this content that I just copied over from the previous Readme.md.  My bad :-(